sources/Renderer/Direct3D11/D3D11SwapChain.cpp

```diff
@@ -286,9 +305,45 @@ void D3D11SwapChain::CreateSwapChain(IDXGIFactory* factory, const Extent2D& reso
         swapChainDesc.Windowed                  = TRUE;//(fullscreen ? FALSE : TRUE);
         swapChainDesc.SwapEffect                = DXGI_SWAP_EFFECT_DISCARD;
     }
-    HRESULT hr = factory->CreateSwapChain(device_.Get(), &swapChainDesc, swapChain_.ReleaseAndGetAddressOf());
+
+    hr = factory->CreateSwapChain(device_.Get(), &swapChainDesc, swapChain_.ReleaseAndGetAddressOf());
```

Comment:
This one fails with `'hr': undeclared identifier` if `LLGL_D3D11_ENABLE_FEATURELEVEL` is 2 or lower (see [AppVeyor CIS job](https://ci.appveyor.com/project/LukasBanana/llgl/builds/49605941/job/3undo0l8qxus4w7e#L257)). Can likely be fixed the same way you fixed it in _D3D11RenderSystem.cpp_.